data/background.ts

```diff
@@ -1,144 +1,391 @@
-  "value": 3
-  }, {
-  "name": "Postsecondary Diploma",
-  "value": 9
+  "value": 4
   }, {
-  "name": "Some postsecondary",
-  "value": 2
-}];
+  "name": "College Diploma",
+  "value": 6
+  }];
 
 
 let ETHNICITY = [{
-  "name": "Hispanic",
-  "value": 1
+  "name": "Middle Eastern Caucasian",
+  "value": 1,
+  "toggle": "ethnicity-all"
 }, {
-  "name": "African",
-  "value": 1
+  "name": "East Asian Caucasian",
+  "value": 1,
+  "toggle": "ethnicity-all"
 }, {
-  "name": "European",
-  "value": 22
+  "name": "Caucasian",
+  "value": 22,
+  "toggle": "ethnicity-all"
 }, {
-  "name": "North American",
-  "value": 27
-  }, {
   "name": "South East Asian",
-  "value": 7
-  }, {
+  "value": 1,
+  "toggle": "ethnicity-all"
+},  {
   "name": "South Asian",
-  "value": 25
+  "value": 8,
+  "toggle": "ethnicity-all"
+}, {
+  "name": "East Asian",
+  "value": 35,
+  "toggle": "ethnicity-all"
 }, {
-"name": "East Asian",
-"value": 55
+  "name": "Middle Eastern",
+  "value": 3,
+  "toggle": "ethnicity-all"
 }, {
-"name": "Middle Eastern",
-"value": 2
+  "name": "Prefer not to disclose",
+  "value": 1,
+  "toggle": "ethnicity-all"
+},
+// Women
+{
+  "name": "Middle Eastern Caucasian",
+  "value": 0,
+  "toggle": "ethnicity-women"
+}, {
+  "name": "East Asian Caucasian",
+  "value": 0,
+  "toggle": "ethnicity-women"
+}, {
+  "name": "Caucasian",
+  "value": 2,
+  "toggle": "ethnicity-women"
+}, {
+  "name": "South East Asian",
+  "value": 0,
+  "toggle": "ethnicity-women"
+},  {
+  "name": "South Asian",
+  "value": 1,
+  "toggle": "ethnicity-women"
+}, {
+  "name": "East Asian",
+  "value": 12,
+  "toggle": "ethnicity-women"
+}, {
+  "name": "Middle Eastern",
+  "value": 0,
+  "toggle": "ethnicity-women"
+}, {
+  "name": "Prefer not to disclose",
+  "value": 0,
+  "toggle": "ethnicity-women"
+},
+// Men
+{
+  "name": "Middle Eastern Caucasian",
+  "value": 1,
+  "toggle": "ethnicity-men"
+}, {
+  "name": "East Asian Caucasian",
+  "value": 1,
+  "toggle": "ethnicity-men"
+}, {
+  "name": "Caucasian",
+  "value": 20,
+  "toggle": "ethnicity-men"
+}, {
+  "name": "South East Asian",
+  "value": 1,
+  "toggle": "ethnicity-men"
+},  {
+  "name": "South Asian",
+  "value": 7,
+  "toggle": "ethnicity-men"
+}, {
+  "name": "East Asian",
+  "value": 21,
+  "toggle": "ethnicity-men"
+}, {
+  "name": "Middle Eastern",
+  "value": 3,
+  "toggle": "ethnicity-men"
+}, {
+  "name": "Prefer not to disclose",
+  "value": 1,
+  "toggle": "ethnicity-men"
 }]
 
 let GENDER = [{
   "name": "Men",
-  "value": 93
+  "value": 55
 }, {
   "name": "Women",
-  "value": 19
+  "value": 15
+}, {
+  "name": "Non-Binary",
+  "value": 1
+}, {
+  "name": "Prefer not to disclose",
+  "value": 1
+}];
+
+let YEAR_OF_BIRTH = [{
+  "name": "1996",
+  "value": 2
+}, {
+  "name": "1997",
+  "value": 9
+}, {
+  "name": "1998",
+  "value": 55
+}, {
+  "name": "1999",
+  "value": 6
+}];
+
+let SEXUAL_ORIENTATION = [{
+  "name": "Heterosexual",
+  "value": 63
+}, {
+  "name": "Homosexual",
```

Comment:
I fear that since there are only 2 entries each for Homosexual and  Bisexual, it might oust people who don't want to be? Can we anonymize this by either
1. Add noise
2. Group both categories together into "non-hetero"?

either way, we then should specify the strategy we used to protect people in the description

Reply:
I don't see how it could out someone, cause it doesn't map LGTBQ+ with any other aspects. So we only know there are 2 homosexual students in our class but not who or any other aspects they share.

If we group the two categories together I would stick with LGBTQ+

Reply:
that's true. It might not oust someone because this data is in isolation from all the other graphs so there's no way to link these 2 students. Adding and disclosing noise might still make it more difficult for students to determine the orientation of people around them. Like you said, we can discuss this tomorrow